commitment_scheme/src/table_prover.rs

```diff
@@ -163,7 +187,8 @@ fn serialize_field_columns<F: PrimeField>(segment: &[Vec<F>]) -> Vec<u8> {
 
     for row in 0..n_rows {
         for col_data in columns.iter().take(n_columns) {
-            serialization.extend_from_slice(&col_data[row].into_bigint().to_bytes_be());
+            let data_mont = col_data[row] * mont_r;
```

Comment:
What is `mont_r`, can it be computed from the field instead of passed?

Reply:
It is the factor used to convert the field element from Standard to Montgomery form. It is used at several places, so just computing it once and passing as argument to other functions.